### PR TITLE
Erc tokens

### PR DESCRIPTION
**Pull request by bergauz**

### CODE REVIEW

**Comment by rkononov** [comment]
LGTM, backend is fine

**Comment by r-gochain** [comment]
@bergauz are we waiting for the UI part?

File: server/backend/tokens.go [hunk 1]
```diff
@@ -12,6 +12,208 @@ import (
...
+	TOKENS_RECEIVED                     InterfaceName = "TOKENS_RECEIVED"
+	TOKENS_TO_SEND                      InterfaceName = "TOKENS_TO_SEND"
+	TOKEN_BY_INDEX                      InterfaceName = "TOKEN_BY_INDEX"
+	TOKEN_OF_OWNER_BY_INDEX             InterfaceName = "TOKEN_OF_OWNER_BY_INDEX"
+	TOKEN_URI                           InterfaceName = "TOKEN_URI"
+	TOTAL_SUPPLY                        InterfaceName = "TOTAL_SUPPLY"
+	TRANSFER                            InterfaceName = "TRANSFER"
+	TRANSFER_1                          InterfaceName = "TRANSFER_1"
+	TRANSFER_2                          InterfaceName = "TRANSFER_2"
+	TRANSFER_AND_CALL                   InterfaceName = "TRANSFER_AND_CALL"
```
Comment (jmank88) [inline server/backend/tokens.go]: Can we use idiomatic go names for these constants? E.g. `TransferFromAndCall` and `ERC20Burnable`.

Also, are the string values used for anything other than map keys? Iota enums would be more efficient

Comment (bergauz) [reply server/backend/tokens.go]: done

File: src/app/scenes/token-asset/token-asset.component.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    /*this._commonService.getAddress(this.contractAddr).pipe(
+      filter((value: Address) => {
+        if (!value || !value.contract || !value.erc_types.includes('Erc721Metadata')) {
+          this._layoutService.offLoading();
+          return false;
+        }
+        return true;
+      }),
+    ).subscribe((value: Address) => {
+    });*/
```
Comment (jmank88) [inline src/app/scenes/token-asset/token-asset.component.ts]: Delete? Or comment why disabled?

File: server/models/address.go [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
+	BalanceString                string          `json:"balance" bson:"balance_string"` //high precise balance for API
+	BalanceWei                   string          `json:"balance_wei" bson:"balance_wei"`
+	UpdatedAt                    time.Time       `json:"updated_at" bson:"updated_at"`
+	UpdatedAtBlock               int64           `json:"-" bson:"updated_at_block"`
+	TokenName                    string          `json:"token_name,omitempty" bson:"token_name"`
+	TokenSymbol                  string          `json:"token_symbol,omitempty" bson:"token_symbol"`
+	Decimals                     int64           `json:"decimals,omitempty" bson:"decimals"`
+	TotalSupply                  string          `json:"total_supply" bson:"total_supply"`
+	Contract                     bool            `json:"contract" bson:"contract"`
+	GO20                         bool            `json:"go20" bson:"go20"`
```
Comment (jmank88) [inline server/models/address.go]: Is this parameter redundant now? Can it be removed? Or at least marked `Deprecated`?

Comment (bergauz) [reply server/models/address.go]: done

File: src/app/scenes/token-asset/token-asset.component.ts (hunk 2 above)
Comment (bergauz) [reply src/app/scenes/token-asset/token-asset.component.ts]: is we will need it in future

**Pull request merged**